src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
@jkotas I'm not sure. ``Compiler::unwindReserveFuncHelper`` seems to pass ``sizeof(func->unwindCodes) - func->unwindCodeSlot;`` as ``unwindSize`` into ``CEEJitInfo::reserveUnwindInfo``.

Is there any documented standard related with x86? I read https://github.com/dotnet/coreclr/blob/master/Documentation/botr/clr-abi.md , but some x86-related stuffs are missing.